src/utp_stream.cpp

```diff
@@ -2444,7 +2409,7 @@ void utp_socket_impl::incoming(std::uint8_t const* buf, int size, packet* p
 	if (!p)
 	{
 		TORRENT_ASSERT(buf);
-		p = static_cast<packet*>(std::malloc(sizeof(packet) + size));
+		p = acquire_packet(std::uint16_t(size));
```

Comment:
again, I don't think this should be an unsigned type. please use ``int`` for the parameter here